numpy/lib/format.py

```diff
@@ -595,7 +595,7 @@ def _read_array_header(fp, version):
         d = safe_eval(header)
     except SyntaxError as e:
         msg = "Cannot parse header: {!r}\nException: {!r}"
-        raise ValueError(msg.format(header, e))
+        raise ValueError(msg.format(header, e)) from e
```

Comment:
I think this one should remove the `Exception: {!r}` bit, as otherwise the exception appears twice. The one that appears to to `from e` appears in more detail, so we should remove the other one.

Reply:
Okay, I'll remove it and update the PR.

Reply:
I've updated PR for this :)